server/types/app.go

```diff
@@ -78,5 +78,14 @@ type (
 
 	// AppExporter is a function that dumps all app state to
 	// JSON-serializable structure and returns the current validator set.
-	AppExporter func(log.Logger, dbm.DB, io.Writer, int64, bool, []string, AppOptions, []string) (ExportedApp, error)
+	AppExporter func(
+		logger log.Logger,
```

Comment:
I only added names for the arguments in this call, to make it slightly easier to understand what an implementation of `AppExporter` should expect.